llpc/lower/llpcSpirvLowerGlobal.cpp

```diff
@@ -348,6 +348,26 @@ void SpirvLowerGlobal::visitCallInst(CallInst &callInst) {
   }
 }
 
+// =====================================================================================================================
+// Check if the given metadata value has a vertex index.
+//
+// @param metaVal : Metadata
+static bool hasVertexIdx(const llvm::Constant &metaVal) {
+  assert(metaVal.getNumOperands() == 4);
+  ShaderInOutMetadata inOutMeta = {};
+  inOutMeta.U64All[0] = cast<ConstantInt>(metaVal.getOperand(2))->getZExtValue();
+  inOutMeta.U64All[1] = cast<ConstantInt>(metaVal.getOperand(3))->getZExtValue();
+
+  if (inOutMeta.IsBuiltIn) {
```

Comment:
According to the suggestion of Jakub, the style of this could be:
```
if (...) {
  return ...
}

return ...
```
Move the return in else branch outside. Make sure we have a return unconditionally in the end the function.